src/EnergyPlus/StandardRatings.cc

```diff
@@ -563,7 +563,19 @@ namespace StandardRatings {
                     // Available chiller capacity as a function of temperature
                     AvailChillerCap = RefCap * ReformEIRChillerCapFT;
 
-                    ReformEIRChillerEIRFPLR = CurveValue(state, EIRFPLRCurveIndex, CondenserOutletTemp, reducedPLR);
+                    switch (state.dataCurveManager->PerfCurve(EIRFPLRCurveIndex)->numDims) {
```

Comment:
You're already grabbing the curve pointer here. Any reason not to save it in an `auto *eirFPLRCurve` and use methods below?

Reply:
Good plan.